zanata-war/src/main/java/org/zanata/action/validator/DuplicateEmailValidator.java

```diff
@@ -34,15 +34,20 @@
 
     @Override
     public boolean isValid(String string, ConstraintValidatorContext context) {
-        if (string == null)
+        if (string == null) {
             return true;
-        if (string.length() == 0)
+        }
+        if (string.length() == 0) {
             return true;
-        PersonDAO personDAO =
-                ServiceLocator.instance().getInstance(PersonDAO.class);
+        }
+        PersonDAO personDAO = getPersonDAO();
```

Comment:
> If you want to refactor little things like this, that's good, but they really need to be in separate commits for reviewability.

Separate from the substantial changes I mean.  Feel free to put multiple minor refactorings in one commit.